### PR TITLE
doc: bluetooth: Fix external web page references

### PR DESCRIPTION
**Pull request by jhedberg**
The generated HTML was broken, even though the rst files themselves contained the correct URLs. Update to a different format which should generate the right output.

### CODE REVIEW

**Comment by jhedberg** [comment]
Disclaimer: I didn't actually test this, rather it's simply what was suggested by @kartben here: https://github.com/zephyrproject-rtos/zephyr/pull/77015#issuecomment-2286414316

File: doc/connectivity/bluetooth/bluetooth-qual.rst [hunk 1]
```diff
@@ -24,11 +24,11 @@ The setup is described in more details in the pages linked below.
...
-.. _bluetooth_qualification_website:
+.. _Bluetooth Qualification website:
    https://qualification.bluetooth.com/
 
 The Zephyr ICS file for the Host features can be downloaded here:
 :download:`ICS_Zephyr_Bluetooth_Host.pts
 </tests/bluetooth/qualification/ICS_Zephyr_Bluetooth_Host.pts>`.
 
-Use the `Bluetooth Qualification website <bluetooth_qualification_website>`_ to view and edit the ICS.
+Use the `_Bluetooth Qualification website`_ to view and edit the ICS.
```
[the remark this answers is not in the document]
Comment (jhedberg) [reply doc/connectivity/bluetooth/bluetooth-qual.rst:34]: I'll wait for the CI docbuild to complete and then check which one needs to be updated.

[the remark this answers is not in the document]
Comment (jori-nordic) [reply doc/connectivity/bluetooth/bluetooth-qual.rst:34]: @theob-pro could you double-check?

[the remark this answers is not in the document]
Comment (jhedberg) [reply doc/connectivity/bluetooth/bluetooth-qual.rst]: Seems like the issue was with this second reference - at least that triggered an error in CI. Let's see now that I pushed a fix.

**Comment by kartben** [comment]
> Disclaimer: I didn't actually test this, rather it's simply what was suggested by @kartben here: [#77015 (comment)](https://github.com/zephyrproject-rtos/zephyr/pull/77015#issuecomment-2286414316)

i didn't test it either, was just showing the general idea, sorry I should have mentioned it upfront :)

**Review by alwa-nordic — APPROVED** [review]

**Review by Thalley — APPROVED** [review]

**Review by theob-pro — APPROVED** [review]

**Review by ubieda — APPROVED** [review]

**Review by kartben — APPROVED** [review]

**Pull request merged**